<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>821</width>
      <height>541</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="clientTab">
     <attribute name="title">
      <string>Client</string>
     </attribute>
     <widget class="QLineEdit" name="clientIpEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>421</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>新細明體</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>192.168.1.1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微軟正黑體</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>IP Address</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="clientProgBar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>461</width>
        <height>51</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="clientStatLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>421</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微軟正黑體</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QPushButton" name="clientConnectBtn">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>150</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微軟正黑體</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>connect</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="clientPortEdit">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>90</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>新細明體</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>8888</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>40</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微軟正黑體</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clientUploadBtn">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>220</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微軟正黑體</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>upload</string>
      </property>
     </widget>
     <widget class="QLabel" name="clientServerInfoLabel">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>290</y>
        <width>301</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微軟正黑體</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Disconnected</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QLabel" name="clientBPSLabel">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>430</y>
        <width>200</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="serverTab">
     <attribute name="title">
      <string>Server</string>
     </attribute>
     <widget class="QProgressBar" name="serverProgBar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>461</width>
        <height>51</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="serverStatLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>451</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微軟正黑體</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="serverPortEdit">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>90</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>新細明體</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>8888</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>40</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微軟正黑體</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微軟正黑體</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>IP Address</string>
      </property>
     </widget>
     <widget class="QPushButton" name="serverListenBtn">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>150</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微軟正黑體</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Listen</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="serverIpBrowser">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>421</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微軟正黑體</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="serverClientInfoLabel">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>290</y>
        <width>241</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微軟正黑體</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Disconnected</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QLabel" name="serverBPSLabel">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>430</y>
        <width>200</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>815</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionRestart"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRestart">
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
